sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -653,3 +654,29 @@ def upgrade_scylla_cluster(self, new_version: str) -> None:
     @staticmethod
     def fstrim_scylla_disks():
         LOGGER.warning("Local K8S backends don't support running 'fstrim' command. Ignoring.")
+
+    def prefill_cluster(self, dataset_name: str):
+        test_data = getattr(datasets, dataset_name, None)
+        if not test_data:
+            raise ValueError(f"Dataset unexpected value: '{dataset_name}'. "
+                             f"Dataset with this name does not defined in the sdcm.utils.sstable.load_inventory")
+
+        test_keyspace_name = 'keyspace1'
+
+        replication_factor = len(self.nodes)
+        node = self.nodes[0]
+        test_keyspaces = self.get_test_keyspaces()
+
+        for i, node in enumerate(self.nodes):
+            # When load a data first time, create schema. if keyspace exists, drop it and create again
+            SstableLoadUtils.upload_sstables(node,
+                                             test_data=test_data[0],
+                                             create_schema=i == 0,
+                                             replication_factor=replication_factor,
+                                             drop_ks_if_exists=test_keyspace_name in test_keyspaces
```

Comment:
this would drop  the key space on each node, if the keyspace existed before

Reply:
No, it will be dropped only if `create_schema == True`

Reply:
changed